src/plugins/zoom.ts

```diff
@@ -66,6 +93,15 @@ class ZoomPlugin extends BasePlugin<ZoomPluginEvents, ZoomPluginOptions> {
     }
     this.container = this.wrapper.parentElement as HTMLElement
     this.wrapper.addEventListener('wheel', this.onWheel)
+
+    // Prevent native scrolling obstructing the zoom on this waveform
+    document.addEventListener('wheel', this.preventScroll, { passive: false });
+
+    if(typeof this.options.maxZoom === 'undefined'){
+      this.options.maxZoom = this.container.clientWidth
+      console.log("maxZoom", this.options.maxZoom)
```

Comment:
Please remove the log and also please run `yarn prettier`.
The code looks good otherwise, cheers!

Reply:
Ok!